api/src/main/java/org/openmrs/PersonAttribute.java

```diff
@@ -42,7 +42,7 @@
  * @see org.openmrs.Attributable
  */
 @Indexed
-public class PersonAttribute extends BaseChangeableOpenmrsData implements java.io.Serializable, Comparable<PersonAttribute> {
+public class PersonAttribute extends BaseAttribute<PersonAttributeType, Person> implements Serializable, Comparable<PersonAttribute> {
```

Comment:
This class probably needs `import org.openmrs.attribute.BaseAttribute` in the imports